packages/grafana-data/src/types/dataLink.ts

```diff
@@ -40,12 +40,20 @@ export interface DataLink<T extends DataQuery = any> {
   internal?: InternalDataLink<T>;
 }
 
+export interface Transformation {
```

Comment:
This needs to be marked as internal.

Reply:
Maybe we can we call it DataLinkTransformationConfig to avoid confusion with DataFrame Transformations?